### ISSUE TITLE
Create demo GDS samples small enough for vignette

### ISSUE THREAD

**Issue by adeschen** [KrasnitzLab/RAIDS#178]
The vignette should have executable examples.

**Comment by mdozmorov** [comment]
In case it helps, here is a script to subset the large snp_pileup files for testing. 
```bash
DIRIN=/Documents/Data/Ancestry/RAIDS/04_snp_pileup
DIROUT=/Documents/Data/Ancestry/RAIDS/data
# Number of random records per chromosome
NUMPERCHR=10 

for file in `find $DIRIN -type f -name "*.txt.gz" | sort | head -n 2`; do
	FILEIN=${file}
	FILEOUT=${DIROUT}/`basename ${FILEIN} .txt.gz`.txt
	echo `basename ${FILEIN}`
	# Save header
	zcat<${FILEIN} | head -n 1 > ${FILEOUT}
	# Unique chromosomes
	CHROMS=`zcat<${FILEIN} | sed '1d' | cut -f1 -d "," | sort -V | uniq`
	# For each chromosome
	for CHR in $CHROMS; do
		# Randomly select NUMPERCHR records
		zcat<${FILEIN} | sed '1d' | grep "^${CHR}," | sort -r | head -n ${NUMPERCHR} | sort >> ${FILEOUT} 
	done
	# Compress
	gzip ${FILEOUT}
done
```
And, a couple of files made with it.
[TEST01.txt.gz](https://github.com/KrasnitzLab/RAIDS/files/9869046/TEST01.txt.gz)
[TEST02.txt.gz](https://github.com/KrasnitzLab/RAIDS/files/9869047/TEST02.txt.gz)

**Comment by adeschen** [comment]
Solved with merge #208
